apps/pdf/util/pdf-puppeteer.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   generate: async(url, destination, tempName) => {
     try {
       const file = `${destination}/${tempName}`;
-      const browser = await puppeteer.launch();
+      const browser = await puppeteer.launch({ args: ['--no-sandbox']});
```

Comment:
This ['--no-sandbox'] argument is discouraged and we should look for an alternative if and when possible